specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/examples/sshPublicKeyExamples/SshPublicKey_GenerateKeyPair.json

```diff
@@ -0,0 +1,17 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "myResourceGroup",
+    "api-version": "2023-09-01",
+    "sshPublicKeyName": "mySshPublicKeyName"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "privateKey": "{ssh private key}",
```

Comment:
> "privateKey": "{ssh private key}", [](http://example.com/codeflow?start=7&length=35)

You need to declare this as x-ms-secret in the swagger, where is that?

Reply:
hi, i am seeing this (https://github.com/Azure/azure-rest-api-specs/blob/bf204aab860f2eb58a9d346b00d44760f2a9b0a2/documentation/Semantic-and-Model-Violations-Reference.md?plain=1#L829) about x-ms-secret, but as you can see, it's being returned here. 
What should we do here?

Reply:
Add a suppression if its only returned via POST calls.